helm/fiftyone-teams-app/values.yaml

```diff
@@ -159,8 +159,8 @@ appSettings:
     # the local cache upon viewing media in the app.
     FIFTYONE_MEDIA_CACHE_APP_IMAGES: false
     # -- Set the media cache size (in bytes) for the local FiftyOne App processes.
-    # The default value is 32 GiB.
-    # `-1` is disabled.
+    # When not set, the app's default value is 32 GiB.
```

Comment:
it feels like it would be pretty difficult to unset this - you'd have to rebake the helm chart?

Not sure there's a lot of value in this note in this context...

Reply:
I think if you set it to `""` we'll get an unexpected behavior

https://github.com/voxel51/fiftyone-teams/blob/main/fiftyone/core/config.py#L756

uses `parse_int` from `eta`: https://github.com/voxel51/eta/blob/main/eta/core/config.py#L832

uses `_parse_env_var_or_key` from `eta`: https://github.com/voxel51/eta/blob/main/eta/core/config.py#L950

Which doesn't seem to handle the "set but empty" case; I think you might get an error...
```
❯ ipython
Python 3.10.13 (main, Feb 13 2024, 23:52:31) [GCC 12.2.0]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.34.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import os

In [2]: os.environ['TEST_VAR'] = ''

In [3]: os.environ.get('TEST_VAR')
Out[3]: ''
```